library/ssl_tls13_keys.h

```diff
@@ -570,4 +570,66 @@ int mbedtls_ssl_tls13_key_schedule_stage_handshake( mbedtls_ssl_context *ssl );
 int mbedtls_ssl_tls13_generate_handshake_keys( mbedtls_ssl_context *ssl,
                                                mbedtls_ssl_key_set *traffic_keys );
 
+/**
+ * \brief Transition into application stage of TLS 1.3 key schedule.
+ *
+ *        The TLS 1.3 key schedule can be viewed as a simple state machine
+ *        with states Initial -> Early -> Handshake -> Application, and
+ *        this function represents the Handshake -> Application transition.
+ *
+ *        In the handshake stage, mbedtls_ssl_tls13_generate_application_keys()
+ *        can be used to derive the handshake traffic keys.
+ *
+ * \param ssl  The SSL context to operate on. This must be in key schedule
+ *             stage \c Handshake.
+ *
+ * \returns    \c 0 on success.
+ * \returns    A negative error code on failure.
+ */
+int mbedtls_ssl_tls13_key_schedule_stage_application(
+    mbedtls_ssl_context *ssl );
```

Comment:
```suggestion
int mbedtls_ssl_tls13_key_schedule_stage_application( mbedtls_ssl_context *ssl );
```

Reply:
I have tried it, it's 85 characters, exceeding 80 characters, so leave it as it is.

Reply:
see above. And double check Coding style document

Reply:
Done.